xl/externals/gi_composites.py

```diff
@@ -152,7 +153,7 @@ class _Child:
         is called
     '''
 
-    __slots__ = []
+    __slots__: List[Gtk.Widget] = []
```

Comment:
Should be `List[str]`, assuming this is the usual Python `__slots__`.

Reply:
Yep.